nf_core/pipeline-template/nextflow_schema.json

```diff
@@ -15,6 +15,7 @@
                 "input": {
                     "type": "string",
                     "format": "file-path",
+                    "exists": true,
                     "mimetype": "text/csv",
                     "pattern": "^\\S+\\.csv$",
                     "schema": "assets/schema_input.json",
```

Comment:
Should we remove this line? 
This will now validate the sample sheet automatically. As we don't have tooling to create a JSON Schema for a sample sheet and this is only optional, the best might be to not have it by default and let users decide if they want to add it back.
```suggestion
```

Reply:
yes, maybe let's remove it for now